Library/ViewModels/ManagePledgeSummaryViewModel.swift

```diff
@@ -38,6 +42,31 @@ public class ManagePledgeSummaryViewModel: ManagePledgeSummaryViewModelType,
     let projectAndBacking = project
       .zip(with: backing)
 
+    let userAndIsBackingProject = backing
+      .filterMap { backing -> (User, Bool)? in
+        guard let user = AppEnvironment.current.currentUser else {
+          return nil
+        }
+
+        return (user, backing.backerId == user.id)
+      }
+
+    self.backerNameLabelHidden = userAndIsBackingProject.map(second).negate()
+    self.circleAvatarViewHidden = userAndIsBackingProject.map(second).negate()
+
+    let userBackingProject = userAndIsBackingProject
+      .filter(second >>> isTrue)
+      .map(first)
+
+    self.backerNameText = userBackingProject
+      .map(\.name)
+
+    self.backerImageURLAndPlaceholderImageName = userBackingProject
+      .map(\.avatar.small)
+      .map(URL.init)
+      .skipNil()
+      .map { ($0, "avatar--placeholder") }
```

Comment:
Wonder if we could get this placeholder to show up in our snapshot tests 🤔

Reply:
Ah! I think maybe we just need to pass the bundle 🤔let me try it

Reply:
Hmm that didn't work. I think it might have something to do with how alamofire animates the placeholder in. We should spend more time looking into it outside of this PR.